modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -204,6 +204,49 @@ baseTest(
 	}
 );
 
+baseTest(
+	'Only 1 Success Popup shows on web content display content edit',
+	{
+		tag: '@LPD-39703',
+	},
+	async ({
+		journalEditArticlePage,
+		page,
+		pagesAdminPage,
+		site,
+		widgetPagePage,
+	}) => {
+		await pagesAdminPage.goto(site.friendlyUrlPath);
+
+		const name = 'widgetPage';
+		await pagesAdminPage.addWidgetPage({name});
+
+		await pagesAdminPage.goto(site.friendlyUrlPath);
+		await page.getByLabel(name, {exact: true}).click();
+
+		await widgetPagePage.addPortlet('Web Content Display');
+
+		await page
+			.getByTestId('addButton')
+			.getByRole('button', {name: 'Add'})
+			.click();
+
+		await page.getByRole('menuitem', {name: 'Basic Web Content'}).click();
+
+		await journalEditArticlePage.fillTitle('testArticle');
+
+		await page.getByRole('button', {name: 'Publish'}).click();
+
+		await expect(page.getByRole('heading', {name})).toBeVisible();
+
+		await page.getByText('Success:Your request').first().isVisible();
+
+		await page.waitForTimeout(100);
+
+		expect(await page.getByText('Success:Your request').count()).toBe(1);
```

Comment:
I leave this comment here because I am quite sure this part will be questioned.
The .first().isVisible() was required because the delay between the first and second Success messages can vary enough so that sometimes it can fail, and sometimes it can pass.
The reason for the wait is the same had a few runs when the whole test finished before the second Success message could popup.
generally, the whole test was tailored in a way that it could consistently detect the number of success messages and fail when required not on race conditions.